packages/storage/amplify_storage_s3/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.3.0
```

Comment:
this should be a patch version update. the change was updating test mocks.

Reply:
The versions are aligned so that if one library bumps to a new minor version, all the libraries are bumped.